test/regressions/index.js

```diff
@@ -119,21 +119,48 @@ const blacklist = [
   'docs-getting-started-templates-dashboard/Orders.png', // Already tested once assembled
   'docs-getting-started-templates-dashboard/Title.png', // Already tested once assembled
   'docs-getting-started-templates-sign-in-side/SignInSide.png', // Flaky
-  'docs-getting-started-usage/Usage.png', // Not needed
-  'docs-guides', // Not needed
-  'docs-styles-advanced', // Not needed
-  'docs-system-borders', // Not needed
-  'docs-system-display', // Not needed
-  'docs-system-flexbox', // Not needed
-  'docs-system-palette', // Not needed
-  'docs-system-positions', // Not needed
-  'docs-system-shadows', // Not needed
-  'docs-system-sizing', // Not needed
-  'docs-system-spacing', // Not needed
-  'docs-system-typography', // Not needed
-  'docs-versions', // Not needed
+  'docs-getting-started-usage/Usage.png',
+  'docs-guides',
+  'docs-styles-advanced',
+  'docs-system-borders',
+  'docs-system-display',
+  'docs-system-flexbox',
+  'docs-system-palette',
+  'docs-system-positions',
+  'docs-system-shadows',
+  'docs-system-sizing',
+  'docs-system-spacing',
+  'docs-system-typography',
+  'docs-versions',
 ];
 
+const unusedBlacklistPatterns = new Set(blacklist);
+
+function excludeTest(suite, name) {
+  if (/^docs-premium-themes(.*)/.test(suite)) {
+    // eslint-disable-next-line no-console
+    console.log('ingoring premium themes pages');
```

Comment:
I know, I know this should return the reason and logging is a different concern but it's friday :)